R/theme61.R

```diff
@@ -241,9 +245,59 @@ theme_e61 <- function(base_size = 14,
   }
 
   return(ret)
+}
+
+
+#' e61 themed graph options in a 'clean' style
+#'
+#' @param base_size Numeric. Chart font size. Default is 14.
+#' @param base_family Character. Chart font family. Default is Arial.
+#'
+#' @return ggplot2 object
+#' @import ggplot2 ggthemes
+#' @export
+#'
+#' @examples
+#' ggplot(data = mtcars, aes(x = wt, y = mpg, col = factor(cyl))) +
+#' geom_point() +
+#' e61_colour_manual(n = 3) +
+#' theme_e61_clean()
 
+theme_e61_clean <- function(
```

Comment:
New functions need to be added to the `_pkgdown.yml` (which I've done for you this time).